### PR TITLE
bootstrap theme for the project

### PR DESCRIPTION
**Pull request by faizankhann**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
+
```
Comment (nauman-lango) [inline app/controllers/application_controller.rb]: Remove this

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<header class="navbar navbar-light navbar-fixed-top navbar-inverse" style="background-color: #e3f2fd;" >
+  <div class="container">
+    <%= link_to "Schedular", '#', id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",   '#' %></li>
+      </ul>
+    </nav>
+  </div>
+</header>
```
Comment (nauman-lango) [inline app/views/layouts/_header.html.erb]: Add new line to end of files

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  #root to: 'home#index'
+  root 'application#hello'
```
Comment (nauman-lango) [inline config/routes.rb]: make root to calendars#index

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (faizankhann) [reply app/views/layouts/_header.html.erb]: Done

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (faizankhann) [reply app/controllers/application_controller.rb]: Done

**Pull request merged**